src/System.Reflection/tests/project.json

```diff
@@ -1,13 +1,16 @@
 {
   "dependencies": {
+    "System.AppContext": "4.0.0",
     "System.Collections": "4.0.10",
     "System.Diagnostics.Debug": "4.0.10",
     "System.Globalization": "4.0.10",
     "System.IO": "4.0.10",
+    "System.IO.FileSystem": "4.0.0.0",
```

Comment:
This should be "4.0.0" (no fourth part) for consistency. I'm surprised this even works. ;)